testgrid/config.yaml

```diff
@@ -4387,6 +4403,26 @@ dashboards:
     base_options: 'exclude-filter-by-regex=^BeforeSuite$'
     alert_options:
       alert_mail_to_addresses: 'gke-kubernetes-accelerators-bugs@google.com'
+  - name: gce-1.10-1.11-gpu-upgrade
+    test_group_name: ci-kubernetes-e2e-gce-stable1-beta-gpu-upgrade
+    base_options: 'exclude-filter-by-regex=^BeforeSuite$'
+    alert_options:
+      alert_mail_to_addresses: 'gke-kubernetes-accelerators-bugs@google.com'
+  - name: gce-1.10-1.12-gpu-upgrade
```

Comment:
I maybe wrong but I don't think Kubernetes is expected to support skip upgrades. You are supposed to upgrade/downgrade one version at a time.

Can you confirm this before adding these tests?

Reply:
you mean master here in stead of 1.12?

Reply:
Good point. I saw there are upgrade tests across multiple versions, like https://k8s-testgrid.appspot.com/sig-cluster-lifecycle-upgrade-skew#gke-gci-1.8-gci-1.10-upgrade-master, but on k8s doc, we do note "that only one minor version upgrade is supported.".

Mostly, I want to test upgrade behavior with master, and the from version can either be beta or stable1. As I understand, beta and master may point to the same minor version depending on where we are in the release cycle, so I chose stable1 here.

@krzyzacy I changed to use master instead of 1.12 here. PTAL.